### ISSUE TITLE
You have too many "likes"

### ISSUE THREAD

**Issue by uncomfyhalomacro** [elkowar/dislike#1]
Your README has a lot of "like"s. delete it.

**Comment by angelofallars** [comment]
I agree with you. I ~~don't like~~ am in disagreement with the decision to put in as many ~~like~~ instances of a four-letter word that starts with the letter L. This is ~~like~~ akin to people asking you to follow the law but breaking them at the same time. ~~Like~~, this is ridiculous. ~~Like~~ give a reaction emoji of approval, comment, and subscribe to me on GitHub if you agree.

**Comment by elkowar** [comment]
I'd, like, love to fix this, but like....
I really don't know how to like,... Phrase a normal sentence anymore

**Comment by mixcoac** [comment]
that's like.. yknow, a really big issue like it should get fixed asap because like, i don't know like i'm not sure.
